marlowe-runtime-web/marlowe-runtime-web.cabal

```diff
@@ -1,25 +1,24 @@
 cabal-version: 3.0
-name: marlowe-runtime-web
-version: 0.0.1
-synopsis:
-  Web server for Marlowe Runtime on Cardano.
-bug-reports: https://github.com/input-output-hk/marlowe-cardano/issues
-license: Apache-2.0
-author: Jamie Bertram
-maintainer: jamie.bertram@iohk.io
-stability: experimental
-category: Language
+name:          marlowe-runtime-web
+version:       0.0.1
+synopsis:      Web server for Marlowe Runtime on Cardano.
+bug-reports:   https://github.com/input-output-hk/marlowe-cardano/issues
+license:       Apache-2.0
+author:        Jamie Bertram
+maintainer:    jamie.bertram@iohk.io
+stability:     experimental
+category:      Language
```

Comment:
Can you please revert the formatting changes?

Reply:
Done.

Reply:
yeah I don't know how this has been formatted...